package/rmt-server.spec

```diff
@@ -180,6 +180,10 @@ rm -f %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/extensions/*/*/*/gem_make.out
 rm -f %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/extensions/*/*/*/mkmf.log
 find %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/gems/yard*/ -type f -exec chmod 644 -- {} +
 
+# lockfile location
+mkdir -p /run/lock/rmt/
+chown -R _rmt /run/lock/rmt/
```

Comment:
Actually, `/run/lock` is owned by `root:lock`, so what we should do is add `_rmt` to the `lock` group.
Then we don't need to `chown` this, and I would make the LOCKFILE_LOCATION simply `/run/lock/rmt.lock` (no `rmt/` subdir)

Reply:
Other than this, this PR LGTM!! :shipit:

Reply:
fixed